include/deal.II/lac/solver_cg.h

```diff
@@ -36,7 +36,9 @@ class PreconditionIdentity;
 /*@{*/
 
 /**
- * Preconditioned cg method for symmetric positive definite matrices. This
+ * his class implements the preconditioned Conjugate Gradients (CG)
+ * method that can be used to solve linear systems with a symmetric positive
+ * definite matrix. This
```

Comment:
It looks a bit strange to break the line here.

Reply:
I sometimes do that because I would have to reindent the whole rest of the paragraph, leading to a much larger patch to review without any actual changes. But happy to do this, of course.